error[E0432]: unresolved import `NonExistent`
  --> $DIR/issue-55457.rs:1:5
   |
LL | use NonExistent; //~ ERROR unresolved import `NonExistent`
   |     ^^^^^^^^^^^ no `NonExistent` external crate

error[E0432]: unresolved import `non_existent`
  --> $DIR/issue-55457.rs:2:5
   |
LL | use non_existent::non_existent; //~ ERROR unresolved import `non_existent`
   |     ^^^^^^^^^^^^ could not find `non_existent` in `{{root}}`

error: cannot determine resolution for the derive macro `NonExistent`
  --> $DIR/issue-55457.rs:5:10
   |
LL | #[derive(NonExistent)] //~ ERROR cannot determine resolution for the derive macro `NonExistent`
   |          ^^^^^^^^^^^
   |
   = note: import resolution is stuck, try simplifying macro imports

error: cannot determine resolution for the attribute macro `non_existent`
  --> $DIR/issue-55457.rs:4:3
   |
LL | #[non_existent] //~ ERROR cannot determine resolution for the attribute macro `non_existent`
   |   ^^^^^^^^^^^^
   |
   = note: import resolution is stuck, try simplifying macro imports

error: aborting due to 4 previous errors

For more information about this error, try `rustc --explain E0432`.
